.gitignore

```diff
@@ -1,8 +1,11 @@
 # IDE
 .idea
 
+fml
 fasttrack
 fasttrack.db*
```

Comment:
do we need the old one?

Reply:
also, maybe we can add more comments here like I did for IDE just do better understand the meaning of this files included into `gitignore`. what do you think?

Reply:
I kept the old one for compatibility, but actually yes, let's just remove it!

I will also add some comment to `.gitignore`, good point!